trackingValidation.py

```diff
@@ -180,32 +282,44 @@ def trackReleaseValidation(filename1, filename2, tag):
     }
 
     histos_nested = collections.defaultdict(lambda: collections.defaultdict(dict))
-    for release in dqm_paths.keys():
+    for label in dqm_paths.keys():
         for coll in trackCollections.values():
             for name in names_nested:
-                histos_nested[release][coll].update({name: dqm_files[release][coll][name].to_hist()})
+                histos_nested[label][coll].update({name: dqm_files[label][coll][name].to_hist()})
 
     for coll in trackCollections.values():
         makedir(os.path.join(plotter.savedir, coll))
         for name, labels in names_nested.items():
-            plotter.plotHistos(h1=histos_nested["pre3"][coll][name],
-                               h2=histos_nested["pre4"][coll][name],
-                               label1="pre3",
-                               label2="pre4",
+            v_histos = [histos_nested[label][coll][name] for label in dqm_paths.keys()]
+            v_labels = [label for label in dqm_paths.keys()]
+            plotter.plotHistos(v_h=v_histos,
+                               v_label=v_labels,
                                ylabel=labels[0],
                                xlabel=labels[1],
                                title=coll,
                                savename=coll + '/' + name)
             
 if __name__ == '__main__':
     parser = argparse.ArgumentParser(description='Make track validation plots.')
-    parser.add_argument('--filename1', type=str, required=True, help='Name of the first ROOT filename.')
-    parser.add_argument('--filename2', type=str, required=True, help='Name of the second ROOT filename.')
-    parser.add_argument('--tag', type=str, required=True, help='Tag to uniquely identify the plots.')
+    parser.add_argument('--files', type=str, required=True, help='Comma-separated list of paths to the ROOT files.')
```

Comment:
you could pass a list (your logic breaks if the strings contain a comma, for instance); ```argparse``` supports lists